godel_robots/abb/godel_irb2400/godel_irb2400_support/launch/irb2400_blending.launch

```diff
@@ -47,10 +49,16 @@
 
   <!-- If the user specifies a 'fake' sensor, publish fake data clouds -->
   <group if="$(arg sim_sensor)">
-    <remap from="generated_cloud" to="sensor_point_cloud"/>
-    <node pkg="godel_surface_detection" type="point_cloud_generator_node" name="point_cloud_generator_node">
-      <rosparam command="load" file="$(find godel_irb2400_support)/config/point_cloud_descriptions.yaml"/>
-    </node>
+    <group if="$(arg real_pcd)">
+      <remap from="cloud_pcd" to="sensor_point_cloud"/>
+      <node pkg="pcl_ros" type="pcd_to_pointcloud" name="point_cloud_generator_node" args="$(arg pcd_location) 0.5" />
```

Comment:
Point clouds will come in different reference frames, so let's set the `frame_id` parameter of this node.